Sources/AsyncHTTPClient/StructuredConcurrencyHelpers.swift

```diff
@@ -0,0 +1,60 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the AsyncHTTPClient open source project
+//
+// Copyright (c) 2025 Apple Inc. and the AsyncHTTPClient project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of AsyncHTTPClient project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+#if compiler(>=6.0)
+@inlinable
+@available(macOS 10.15, iOS 13.0, watchOS 6.0, tvOS 13.0, *)
+internal func asyncDo<R>(
+    isolation: isolated (any Actor)? = #isolation,
+    _ body: sending () async throws -> sending R,
```

Comment:
```suggestion
    _ body: () async throws -> sending R,
```

Reply:
Alright, I tightened them all down to what I think is the minimum in the current implementation. Feel slightly uneasy about tightening it so much as it might lock us in. But I'm okay going with what you use as the standard shape.